app/server.go

```diff
@@ -1182,6 +1182,12 @@ func (s *Server) startLocalModeServer() error {
 	}
 
 	socket := *s.configStore.Get().ServiceSettings.LocalModeSocketLocation
+	if _, err := os.Stat(socket); !os.IsNotExist(err) {
+		if err = os.Remove(socket); err != nil {
```

Comment:
Why not use `os.RemoveAll`? Then there is no need to check for anything.

Reply:
Is it any different for a single file?

Reply:
Yes, `RemoveAll` will not throw an error if the file doesn't exist.